package.json

```diff
@@ -23,6 +23,7 @@
     "prettier": "node scripts/prettier.js write",
     "tck": "yarn run build && node packages/rsocket-tck/build/index.js \"$@\"",
     "simple-cli": "yarn run build && node packages/rsocket-examples/build/SimpleCli.js \"$@\"",
+    "resume-example": "yarn run build && node packages/rsocket-examples/build/ResumeExample.js",
```

Comment:
this was ran against simple java server with proxy between client and server, to test resumability `socat -d TCP-LISTEN:5001,fork,reuseaddr TCP:localhost:5000`